Formula/d/docker-completion.rb

```diff
@@ -2,7 +2,7 @@ class DockerCompletion < Formula
   desc "Bash, Zsh and Fish completion for Docker"
   homepage "https://www.docker.com/"
   url "https://github.com/docker/cli.git",
-      tag:      "v25.0.1",
+      tag:      "v25.0.2",
```

Comment:
it is the same revision but with diff tag?

Reply:
yea strange, might have been a mistake

Reply:
The revision for v25.0.1 is 293681613032e6d1a39cc88115847d3984195c24. Wonder if there was retagging. This should be fine to merge given that's the actual revision for v25.0.2.

Reply:
okay, merging.